### ISSUE TITLE
MEGASync right click doesn't work proper

### ISSUE THREAD

**Issue by krieghof** [martinpl/tray-icons-reloaded#6]
When I right click on the Mega try Icon, the right click menu pop open, and it also opens the application itself.

Hope you can fix this, because you made a really great extension there, keep up the good work.

**Comment by krieghof** [comment]
It seems to work correctly on Gnome 3.36 I had this problem on 3.34

**Comment by martinpl** [comment]
Definitely app bug, i send patch for it but may not be accepted. 
Still have it on Arch 3.36.1.

https://github.com/meganz/MEGAsync/issues/164

**Comment by krieghof** [comment]
Hmm, weird I'm on popOs 20.04 and it works. 
Any idea why it works on some distros and on some not?

**Comment by martinpl** [comment]
Any chance that with 20.04 update KStatusNotifierItem/AppIndicator has been install / enabled?

**Comment by krieghof** [comment]
Yeah you are right, pop installed KStatusNotifierItem/AppIndicator, the bug is still present
